modules/projects/client/controllers/project.client.controllers.js

```diff
@@ -289,7 +289,7 @@ function controllerProjectEntry ($scope, $state, $stateParams, $uibModal, projec
           .then(function(data) {
             $scope.project = data;
             AlertService.success('Project was added.', 4000);
-            $scope.goToEdit(currTab);
+            $state.go('p.edit', { projectid:  $scope.project.code, currTab: currTab }, { reload: true });
```

Comment:
Something funny happening with the merge.  These look like other peoples changes. Here and a bunch of the changes below.